src/pages/settings/Security/CloseAccountPage.js

```diff
@@ -146,9 +152,9 @@ export default compose(
     withLocalize,
     withWindowDimensions,
     withOnyx({
-        isCloseAccoutModalOpen: {
-            key: ONYXKEYS.IS_CLOSE_ACCOUNT_MODAL_OPEN,
-            initWithStoredValues: false,
+        closeAccount: {
+            key: ONYXKEYS.CLOSE_ACCOUNT,
+            initWithStoredValues: {error: '', isLoading: false},
```

Comment:
This is actually expecting a boolean and not the values

https://github.com/Expensify/react-native-onyx/blob/main/API.md#connectmapping--number

Reply:
ohhhhh interesting, good catch. It looks like in that repo we default `initWithStoredValues` to `true` so I think I can pretty much just delete this line - we do actually want to load the page with the error / loading state stored